aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -1,8 +1,19 @@
 from dataclasses import dataclass
+from typing import Any
+
 
 @dataclass
 class ReviewComment:
     file_path: str
     line: int
```

Comment:
**style**: The type field accepts any string value. Consider using an Enum class to restrict possible values to the documented options (naming, style, indentation, documentation, test_coverage).